### DISCUSSION TITLE
Is it possible to build open apple operating systems using bsd licence like ravynos for alternative macos?

### DISCUSSION THREAD

**Discussion by stevexuzeta** [ravynsoft/ravynos#417] [General]
Can we use bsd or apache or mit licences to build the open apple operating systems. For example, use freebsd and cocoa api to build desktop os for alternative  macos like ravynos, use freebsd and uikit api to build a phone os for alternative  ios, and also watchos and visionos and so on.

**Comment by mszoek** [comment]
Maybe I don't understand this question. I think this describes what we're doing.

**Reply by stevexuzeta** [reply]
That's so great, thank you very much for your contribution. As far as I know, RavynOS is a replacement for macOS, which is a desktop operating system for PC. Do you have any other projects or plans to create replacements of other Apple operating systems such as iOS, WatchOS, and VisionOS?

**Reply by mszoek** [reply]
I'm not aware of any projects like those yet. My focus will be on core macOS features until it is mostly compatible - no plans to work on other platforms for now.

**Reply by stevexuzeta** [reply]
All right. Thank you for your detailed explanation. Once again, thank you for your great contribution to RavynOS.

**Reply by upintheairsheep** [reply]
Ravynos will likely have different desktop environments for replacing other OSs when they get implemented
